app/lib/foreman_deployments/tasks/creation_task_definition.rb

```diff
@@ -29,13 +30,26 @@ def self.create_object(parameters)
         object_params = parameters['params']
 
         object_type = object_type.constantize if object_type.is_a? String
-        object_type.new(object_params)
+        object = object_type.new
+        object_params.each do |key, value|
+          begin
+            object.attributes = { key => value }
```

Comment:
It doesn't, see:
http://apidock.com/rails/ActiveRecord/Base/attributes%3D
http://apidock.com/rails/ActiveRecord/Base/assign_attributes
But there's actually no benefit in using `attributes=` over the `send("#{key}="...` so I'll use send.